images/capi/packer/config/common.json

```diff
@@ -10,5 +10,6 @@
   "pause_image": "k8s.gcr.io/pause:3.6",
   "redhat_epel_rpm": "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm",
   "reenable_public_repos": "true",
-  "remove_extra_repos": "false"
+  "remove_extra_repos": "false",
+  "ssh_timeout": "4h"
```

Comment:
ssh_timeout is not very common and different for all targets (ova, qemu, raw, ...). Therefore I'd wouldn't set it here.

Reply:
good point, i've pushed it to the qemu packer.json file